cmd/commands/cli/command.go

```diff
@@ -356,6 +357,47 @@ func (c *cliApp) unlock(argsString string) {
 	success(fmt.Sprintf("Identity %s unlocked.", identity))
 }
 
+func (c *cliApp) payout(argsString string) {
+	args := strings.Fields(argsString)
+
+	const usage = "payout command:\n    set"
+	if len(args) < 1 {
```

Comment:
I think this is more explicit and clear:

```suggestion
	if len(args) == 0 {
```

Reply:
`+`